graphql/schemabuilder/types.go

```diff
@@ -213,7 +267,7 @@ type method struct {
 	TextFilterFuncs map[string]Filter
 
 	// Sort methods
-	SortFuncs map[string]interface{}
+	SortFuncs map[string]Sort
```

Comment:
Maybe consider renaming this: it's a little misleading to call this `SortFuncs` when it actually points to a `Sort` structure rather than the function.